tests/models/test_tpu.py

```diff
@@ -237,40 +246,41 @@ def test_dataloaders_passed_to_fit(tmpdir):
     tutils.reset_seed()
     model = BoringModel()
 
-    trainer = Trainer(default_root_dir=tmpdir, max_epochs=1, tpu_cores=8)
+    trainer = Trainer(default_root_dir=tmpdir, max_epochs=1, accelerator="tpu", devices=8)
     trainer.fit(model, train_dataloaders=model.train_dataloader(), val_dataloaders=model.val_dataloader())
     assert trainer.state.finished, f"Training failed with {trainer.state}"
 
 
 @pytest.mark.parametrize(
-    ["tpu_cores", "expected_tpu_id"],
+    ["devices", "expected_tpu_id"],
     [(1, None), (8, None), ([1], 1), ([8], 8)],
 )
 @RunIf(tpu=True)
-def test_tpu_id_to_be_as_expected(tpu_cores, expected_tpu_id):
+def test_tpu_id_to_be_as_expected(devices, expected_tpu_id):
     """Test if trainer.tpu_id is set as expected."""
-    assert Trainer(tpu_cores=tpu_cores)._accelerator_connector.tpu_id == expected_tpu_id
+    assert Trainer(accelerator="tpu", devices=devices)._accelerator_connector.tpu_id == expected_tpu_id
 
 
+@RunIf(tpu=True)
 def test_tpu_misconfiguration():
     """Test if trainer.tpu_id is set as expected."""
     with pytest.raises(MisconfigurationException, match="`tpu_cores` can only be"):
-        Trainer(tpu_cores=[1, 8])
+        Trainer(accelerator="tpu", devices=[1, 8])
 
 
 @pytest.mark.skipif(_TPU_AVAILABLE, reason="test requires missing TPU")
 def test_exception_when_no_tpu_found(tmpdir):
     """Test if exception is thrown when xla devices are not available."""
 
-    with pytest.raises(MisconfigurationException, match="No TPU devices were found."):
-        Trainer(tpu_cores=8)
+    with pytest.raises(MisconfigurationException, match="but TPUs are not available."):
+        Trainer(accelerator="tpu", devices=8)
```

Comment:
The test case here is failing, and should be fixed via #12047.

Reply:
Thanks for letting me know!